model/riscv_fdext_regs.sail

```diff
@@ -270,30 +270,48 @@ function wF_bits(i: bits(5), data: flenbits) -> unit = {
 
 overload F = {rF_bits, wF_bits, rF, wF}
 
+val rF_H : bits(5) -> bits(16) effect {escape, rreg}
+function rF_H(i) = {
+  assert(sizeof(flen) >= 16);
```

Comment:
Should probably also assert !zfinx for completeness (especially to catch things like someone adding Zfinx+Zfa but forgetting to update one of the accesses to the right function and trying to debug what on earth went wrong)